src/main/scala/mesosphere/marathon/state/UnreachableStrategy.scala

```diff
@@ -34,13 +32,27 @@ object UnreachableEnabled {
   val DefaultExpungeAfter: FiniteDuration = 0.seconds
   val default = UnreachableEnabled()
 
-  implicit val unreachableEnabledValidator = validator[UnreachableEnabled] { strategy =>
-    strategy.inactiveAfter should be >= 0.second
-    strategy.inactiveAfter should be <= strategy.expungeAfter
+  implicit val unreachableEnabledValidator = new Validator[UnreachableEnabled] {
+    override def apply(unreachableEnabled: UnreachableEnabled): Result = {
+      if (unreachableEnabled.inactiveAfter < 0.seconds)
+        Failure(Set(
+          RuleViolation(unreachableEnabled.inactiveAfter, UnreachableStrategy.InactiveAfterGreaterThanZeroMessage, Path(Generic("inactiveAfterSeconds")))))
+      else if (unreachableEnabled.inactiveAfter > unreachableEnabled.expungeAfter)
+        Failure(Set(
+          RuleViolation(
+            unreachableEnabled.inactiveAfter,
+            UnreachableStrategy.inactiveAfterSmallerThanExpungeAfterMessage(unreachableEnabled.expungeAfter),
+            Path(Generic("inactiveAfterSeconds")))))
+      else
+        Success
+    }
   }
 }
 
 object UnreachableStrategy {
+  val InactiveAfterGreaterThanZeroMessage = "inactiveAfterSeconds must be greater than or equal to 0"
+  def inactiveAfterSmallerThanExpungeAfterMessage(expungeAfter: Duration) =
+    s"inactiveAfterSeconds must be less or equal to expungeAfterSeconds, which is ${expungeAfter.toSeconds}"
```

Comment:
I think we should log both values:

`inactiveAfterSeconds (10s) must be less or equal to expungeAfterSeconds (5s)`

Reply:
I really think we should add the times.